xds/internal/clusterspecifier/rls/rls.go

```diff
@@ -40,6 +40,24 @@ func init() {
 	}
 }
 
+// RegisterForTesting registers the RLS Cluster Specifier Plugin for testing
+// purposes, regardless of the XDSRLS environment variable. This is needed
+// because there is no way to set the XDSRLS environment variable to true in a
+// test before init() in this package is run.
```

Comment:
Does this mean that we can get rid of this once the env var is removed? If so, could you please add a TODO for the same here.